src/settings.ts

```diff
@@ -46,6 +46,9 @@ export const DEFAULT_SETTINGS: GrammarlySettings = {
     '[mdx]': {
       ignore: ['code'],
     },
+    '[asciidoc]': {
+      ignore: ['code'],
+    },
```

Comment:
Is it possible to just remove the `ignore` functionality so that asciidoctor files are (at least minimally) supported?

```suggestion
```